src/views/EditDetails.js

```diff
@@ -0,0 +1,85 @@
+import axios from 'axios'
+import React from 'react'
+import config from '../config'
+import FormFieldRow from '../components/FormFieldRow'
+import ErrorHandler from '../components/ErrorHandler'
+
+const emailValidRegex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
We might collect our our regular expressions in one file and reuse them from there.